fineract-provider/config/swagger/config.json

```diff
@@ -0,0 +1,12 @@
+{
+   "library": "retrofit2",
+   "modelPackage": "org.mifos.models",
```

Comment:
We shouldn't use "org.mifos" for Java package names that are code generated in an Apache project... how about making this e.g. org.apache.fineract.client?

Reply:
Sorry about that! I missed it out. I'll change it to **org.apache.fineract.client**

To verify the changes:
- "modelPackage": "org.apache.fineract.client.models"
- "apiPackage": "org.apache.fineract.client.services"
- "invokerPackage": "org.apache.fineract.client"
- "groupId": "org.apache.fineract"
- "artifactId": "client"

This is how we want right?

> could you put the description about how to use this which you provided above into the/README.md as part of this PR?
Yeah sure. I'll add this in the Readme.md